src/button.tsx

```diff
@@ -377,7 +384,7 @@ const Button = React.forwardRef<TouchableElement, ButtonProps & {type: ButtonTyp
             EndIcon: props.EndIcon,
         }),
         disabled: props.disabled || showSpinner || isFormSending,
-        role: 'button',
+        role: props.role,
```

Comment:
Shouldn't we default to button in case role was not provided?

Reply:
the default is provided by the dom element (button for `<button>` and link for `<a>`)